src/js/stores/MesosSummaryStore.js

```diff
@@ -169,7 +180,9 @@ class MesosSummaryStore extends GetSetBaseStore {
     let states = this.get('states');
 
     if (typeof data.date !== 'number') {
-      data.date = Date.now();
+      let lastRequestTime = this.getNextRequestTime();
```

Comment:
Isn't this going to return the next request time or the current time if it's the first time this gets called?